[CI] DocsClientYamlTestSuiteIT class failing

**Build Scans:**
- [elasticsearch-periodic-platform-support #5367 / windows-2019_checkpart1_platform-support-windows](https://gradle-enterprise.elastic.co/s/55zrhpkmrr3ru)
- [elasticsearch-pull-request #46522 / docs-check](https://gradle-enterprise.elastic.co/s/7wbom24xfxmyc)

**Reproduction Line:**
```
gradlew ":docs:yamlRestTest" --tests "org.elasticsearch.smoketest.DocsClientYamlTestSuiteIT.test {yaml=reference/eql/eql/line_141}" -Dtests.seed=AF521FA4078A9469 -Dtests.locale=su-ID -Dtests.timezone=Europe/San_Marino -Druntime.java=23
```

**Applicable branches:**
8.16

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.smoketest.DocsClientYamlTestSuiteIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!(org.elasticsearch.smoketest.DocsClientYamlTestSuiteIT),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: Failure at [reference/eql/eql:7]: expected [2xx] status code but api [raw[method=DELETE path=_index_template/*]] returned [400 Bad Request] [{"error":{"root_cause":[{"type":"illegal_argument_exception","reason":"unable to remove composable templates [.deprecation-indexing-template, .fleet-fileds-fromhost-data, .fleet-fileds-fromhost-meta, .fleet-fileds-tohost-data, .fleet-fileds-tohost-meta, .kibana-reporting, .ml-anomalies-, .ml-notifications-000002, .ml-state, .ml-stats, .monitoring-beats-mb, .monitoring-ent-search-mb, .monitoring-es-mb, .monitoring-kibana-mb, .monitoring-logstash-mb, .slm-history-7, .watch-history-16, behavioral_analytics-events-default, elastic-connectors, elastic-connectors-sync-jobs, logs, logs-apm.app@template, logs-apm.error@template, logs-otel@template, metrics, metrics-apm.app@template, metrics-apm.internal@template, metrics-apm.service_destination.10m@template, metrics-apm.service_destination.1m@template, metrics-apm.service_destination.60m@template, metrics-apm.service_summary.10m@template, metrics-apm.service_summary.1m@template, metrics-apm.service_summary.60m@template, metrics-apm.service_transaction.10m@template, metrics-apm.service_transaction.1m@template, metrics-apm.service_transaction.60m@template, metrics-apm.transaction.10m@template, metrics-apm.transaction.1m@template, metrics-apm.transaction.60m@template, metrics-otel@template, metrics-service_destination.10m@template, metrics-service_destination.1m@template, metrics-service_destination.60m@template, metrics-service_summary.10m.otel@template, metrics-service_summary.1m.otel@template, metrics-service_summary.60m.otel@template, metrics-service_transaction.10m.otel@template, metrics-service_transaction.1m.otel@template, metrics-service_transaction.60m.otel@template, metrics-transaction.10m.otel@template, metrics-transaction.1m.otel@template, metrics-transaction.60m.otel@template, my-data-stream-template, search-acl-filter, synthetics, traces-apm.rum@template, traces-apm.sampled@template, traces-apm@template, traces-otel@template] as they are in use by a data streams [.slm-history-7]","stack_trace":"org.elasticsearch.ElasticsearchException$1: unable to remove composable templates [.deprecation-indexing-template, .fleet-fileds-fromhost-data, .fleet-fileds-fromhost-meta, .fleet-fileds-tohost-data, .fleet-fileds-tohost-meta, .kibana-reporting, .ml-anomalies-, .ml-notifications-000002, .ml-state, .ml-stats, .monitoring-beats-mb, .monitoring-ent-search-mb, .monitoring-es-mb, .monitoring-kibana-mb, .monitoring-logstash-mb, .slm-history-7, .watch-history-16, behavioral_analytics-events-default, elastic-connectors, elastic-connectors-sync-jobs, logs, logs-apm.app@template, logs-apm.error@template, logs-otel@template, metrics, metrics-apm.app@template, metrics-apm.internal@template, metrics-apm.service_destination.10m@template, metrics-apm.service_destination.1m@template, metrics-apm.service_destination.60m@template, metrics-apm.service_summary.10m@template, metrics-apm.service_summary.1m@template, metrics-apm.service_summary.60m@template, metrics-apm.service_transaction.10m@template, metrics-apm.service_transaction.1m@template, metrics-apm.service_transaction.60m@template, metrics-apm.transaction.10m@template, metrics-apm.transaction.1m@template, metrics-apm.transaction.60m@template, metrics-otel@template, metrics-service_destination.10m@template, metrics-service_destination.1m@template, metrics-service_destination.60m@template, metrics-service_summary.10m.otel@template, metrics-service_summary.1m.otel@template, metrics-service_summary.60m.otel@template, metrics-service_transaction.10m.otel@template, metrics-service_transaction.1m.otel@template, metrics-service_transaction.60m.otel@template, metrics-transaction.10m.otel@template, metrics-transaction.1m.otel@template, metrics-transaction.60m.otel@template, my-data-stream-template, search-acl-filter, synthetics, traces-apm.rum@template, traces-apm.sampled@template, traces-apm@template, traces-otel@template] as they are in use by a data streams [.slm-history-7]\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.ElasticsearchException.guessRootCauses(ElasticsearchException.java:706)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.ElasticsearchException.generateFailureXContent(ElasticsearchException.java:634)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.rest.RestResponse.build(RestResponse.java:187)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.rest.RestResponse.<init>(RestResponse.java:146)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.rest.RestResponse.<init>(RestResponse.java:117)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.rest.action.RestActionListener.onFailure(RestActionListener.java:54)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.action.ActionListenerImplementations.safeAcceptException(ActionListenerImplementations.java:64)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.action.ActionListenerImplementations.safeOnFailure(ActionListenerImplementations.java:75)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.action.ActionListener$3.onFailure(ActionListener.java:410)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.tasks.TaskManager$1.onFailure(TaskManager.java:215)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.action.ActionListenerImplementations.safeAcceptException(ActionListenerImplementations.java:64)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.action.ActionListenerImplementations.safeOnFailure(ActionListenerImplementations.java:75)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.action.DelegatingActionListener.onFailure(DelegatingActionListener.java:32)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.action.ActionListenerImplementations$RunBeforeActionListener.onFailure(ActionListenerImplementations.java:346)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.action.ActionListenerImplementations.safeAcceptException(ActionListenerImplementations.java:64)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.action.ActionListenerImplementations.safeOnFailure(ActionListenerImplementations.java:75)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.action.ActionListener$3.onFailure(ActionListener.java:410)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.action.ActionListenerImplementations.safeAcceptException(ActionListenerImplementations.java:64)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.action.ActionListenerImplementations.safeOnFailure(ActionListenerImplementations.java:75)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.action.DelegatingActionListener.onFailure(DelegatingActionListener.java:32)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.action.support.ContextPreservingActionListener.onFailure(ContextPreservingActionListener.java:40)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.action.ActionListenerImplementations.safeAcceptException(ActionListenerImplementations.java:64)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.action.ActionListenerImplementations.safeOnFailure(ActionListenerImplementations.java:75)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.action.DelegatingActionListener.onFailure(DelegatingActionListener.java:32)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.action.ActionListenerImplementations.safeAcceptException(ActionListenerImplementations.java:64)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.action.ActionListenerImplementations.safeOnFailure(ActionListenerImplementations.java:75)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.action.DelegatingActionListener.onFailure(DelegatingActionListener.java:32)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.action.ActionListenerImplementations$RunBeforeActionListener.onFailure(ActionListenerImplementations.java:346)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.action.ActionListenerImplementations.safeAcceptException(ActionListenerImplementations.java:64)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.action.ActionListenerImplementations.safeOnFailure(ActionListenerImplementations.java:75)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.action.ActionListener$3.onFailure(ActionListener.java:410)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.action.support.master.TransportMasterNodeAction$AsyncSingleAction.lambda$doStart$2(TransportMasterNodeAction.java:235)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.action.ActionListenerImplementations$DelegatingResponseActionListener.acceptException(ActionListenerImplementations.java:188)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.action.ActionListenerImplementations.safeAcceptException(ActionListenerImplementations.java:64)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.action.ActionListenerImplementations$DelegatingResponseActionListener.onFailure(ActionListenerImplementations.java:193)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.cluster.metadata.MetadataIndexTemplateService$TemplateClusterStateUpdateTask.onFailure(MetadataIndexTemplateService.java:174)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.cluster.service.MasterService$ExecutionResult.notifyFailure(MasterService.java:1009)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.cluster.service.MasterService$ExecutionResult.onClusterStateUnchanged(MasterService.java:972)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.cluster.service.MasterService.executeAndPublishBatch(MasterService.java:258)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.cluster.service.MasterService$BatchingTaskQueue$Processor.lambda$run$2(MasterService.java:1691)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.action.ActionListener.run(ActionListener.java:452)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.cluster.service.MasterService$BatchingTaskQueue$Processor.run(MasterService.java:1688)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.cluster.service.MasterService$5.lambda$doRun$0(MasterService.java:1283)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.action.ActionListener.run(ActionListener.java:452)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.cluster.service.MasterService$5.doRun(MasterService.java:1262)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:1023)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:27)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\r\n\tat java.base/java.lang.Thread.run(Thread.java:1575)\r\nCaused by: java.lang.IllegalArgumentException: unable to remove composable templates [.deprecation-indexing-template, .fleet-fileds-fromhost-data, .fleet-fileds-fromhost-meta, .fleet-fileds-tohost-data, .fleet-fileds-tohost-meta, .kibana-reporting, .ml-anomalies-, .ml-notifications-000002, .ml-state, .ml-stats, .monitoring-beats-mb, .monitoring-ent-search-mb, .monitoring-es-mb, .monitoring-kibana-mb, .monitoring-logstash-mb, .slm-history-7, .watch-history-16, behavioral_analytics-events-default, elastic-connectors, elastic-connectors-sync-jobs, logs, logs-apm.app@template, logs-apm.error@template, logs-otel@template, metrics, metrics-apm.app@template, metrics-apm.internal@template, metrics-apm.service_destination.10m@template, metrics-apm.service_destination.1m@template, metrics-apm.service_destination.60m@template, metrics-apm.service_summary.10m@template, metrics-apm.service_summary.1m@template, metrics-apm.service_summary.60m@template, metrics-apm.service_transaction.10m@template, metrics-apm.service_transaction.1m@template, metrics-apm.service_transaction.60m@template, metrics-apm.transaction.10m@template, metrics-apm.transaction.1m@template, metrics-apm.transaction.60m@template, metrics-otel@template, metrics-service_destination.10m@template, metrics-service_destination.1m@template, metrics-service_destination.60m@template, metrics-service_summary.10m.otel@template, metrics-service_summary.1m.otel@template, metrics-service_summary.60m.otel@template, metrics-service_transaction.10m.otel@template, metrics-service_transaction.1m.otel@template, metrics-service_transaction.60m.otel@template, metrics-transaction.10m.otel@template, metrics-transaction.1m.otel@template, metrics-transaction.60m.otel@template, my-data-stream-template, search-acl-filter, synthetics, traces-apm.rum@template, traces-apm.sampled@template, traces-apm@template, traces-otel@template] as they are in use by a data streams [.slm-history-7]\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.cluster.metadata.MetadataIndexTemplateService.innerRemoveIndexTemplateV2(MetadataIndexTemplateService.java:1023)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.cluster.metadata.MetadataIndexTemplateService$6.execute(MetadataIndexTemplateService.java:972)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.cluster.metadata.MetadataIndexTemplateService$1.executeTask(MetadataIndexTemplateService.java:150)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.cluster.metadata.MetadataIndexTemplateService$1.executeTask(MetadataIndexTemplateService.java:147)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.cluster.SimpleBatchedExecutor.execute(SimpleBatchedExecutor.java:71)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.cluster.service.MasterService.innerExecuteTasks(MasterService.java:1075)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.cluster.service.MasterService.executeTasks(MasterService.java:1038)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.cluster.service.MasterService.executeAndPublishBatch(MasterService.java:245)\r\n\t... 11 more\r\n"}],"type":"illegal_argument_exception","reason":"unable to remove composable templates [.deprecation-indexing-template, .fleet-fileds-fromhost-data, .fleet-fileds-fromhost-meta, .fleet-fileds-tohost-data, .fleet-fileds-tohost-meta, .kibana-reporting, .ml-anomalies-, .ml-notifications-000002, .ml-state, .ml-stats, .monitoring-beats-mb, .monitoring-ent-search-mb, .monitoring-es-mb, .monitoring-kibana-mb, .monitoring-logstash-mb, .slm-history-7, .watch-history-16, behavioral_analytics-events-default, elastic-connectors, elastic-connectors-sync-jobs, logs, logs-apm.app@template, logs-apm.error@template, logs-otel@template, metrics, metrics-apm.app@template, metrics-apm.internal@template, metrics-apm.service_destination.10m@template, metrics-apm.service_destination.1m@template, metrics-apm.service_destination.60m@template, metrics-apm.service_summary.10m@template, metrics-apm.service_summary.1m@template, metrics-apm.service_summary.60m@template, metrics-apm.service_transaction.10m@template, metrics-apm.service_transaction.1m@template, metrics-apm.service_transaction.60m@template, metrics-apm.transaction.10m@template, metrics-apm.transaction.1m@template, metrics-apm.transaction.60m@template, metrics-otel@template, metrics-service_destination.10m@template, metrics-service_destination.1m@template, metrics-service_destination.60m@template, metrics-service_summary.10m.otel@template, metrics-service_summary.1m.otel@template, metrics-service_summary.60m.otel@template, metrics-service_transaction.10m.otel@template, metrics-service_transaction.1m.otel@template, metrics-service_transaction.60m.otel@template, metrics-transaction.10m.otel@template, metrics-transaction.1m.otel@template, metrics-transaction.60m.otel@template, my-data-stream-template, search-acl-filter, synthetics, traces-apm.rum@template, traces-apm.sampled@template, traces-apm@template, traces-otel@template] as they are in use by a data streams [.slm-history-7]","stack_trace":"java.lang.IllegalArgumentException: unable to remove composable templates [.deprecation-indexing-template, .fleet-fileds-fromhost-data, .fleet-fileds-fromhost-meta, .fleet-fileds-tohost-data, .fleet-fileds-tohost-meta, .kibana-reporting, .ml-anomalies-, .ml-notifications-000002, .ml-state, .ml-stats, .monitoring-beats-mb, .monitoring-ent-search-mb, .monitoring-es-mb, .monitoring-kibana-mb, .monitoring-logstash-mb, .slm-history-7, .watch-history-16, behavioral_analytics-events-default, elastic-connectors, elastic-connectors-sync-jobs, logs, logs-apm.app@template, logs-apm.error@template, logs-otel@template, metrics, metrics-apm.app@template, metrics-apm.internal@template, metrics-apm.service_destination.10m@template, metrics-apm.service_destination.1m@template, metrics-apm.service_destination.60m@template, metrics-apm.service_summary.10m@template, metrics-apm.service_summary.1m@template, metrics-apm.service_summary.60m@template, metrics-apm.service_transaction.10m@template, metrics-apm.service_transaction.1m@template, metrics-apm.service_transaction.60m@template, metrics-apm.transaction.10m@template, metrics-apm.transaction.1m@template, metrics-apm.transaction.60m@template, metrics-otel@template, metrics-service_destination.10m@template, metrics-service_destination.1m@template, metrics-service_destination.60m@template, metrics-service_summary.10m.otel@template, metrics-service_summary.1m.otel@template, metrics-service_summary.60m.otel@template, metrics-service_transaction.10m.otel@template, metrics-service_transaction.1m.otel@template, metrics-service_transaction.60m.otel@template, metrics-transaction.10m.otel@template, metrics-transaction.1m.otel@template, metrics-transaction.60m.otel@template, my-data-stream-template, search-acl-filter, synthetics, traces-apm.rum@template, traces-apm.sampled@template, traces-apm@template, traces-otel@template] as they are in use by a data streams [.slm-history-7]\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.cluster.metadata.MetadataIndexTemplateService.innerRemoveIndexTemplateV2(MetadataIndexTemplateService.java:1023)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.cluster.metadata.MetadataIndexTemplateService$6.execute(MetadataIndexTemplateService.java:972)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.cluster.metadata.MetadataIndexTemplateService$1.executeTask(MetadataIndexTemplateService.java:150)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.cluster.metadata.MetadataIndexTemplateService$1.executeTask(MetadataIndexTemplateService.java:147)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.cluster.SimpleBatchedExecutor.execute(SimpleBatchedExecutor.java:71)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.cluster.service.MasterService.innerExecuteTasks(MasterService.java:1075)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.cluster.service.MasterService.executeTasks(MasterService.java:1038)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.cluster.service.MasterService.executeAndPublishBatch(MasterService.java:245)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.cluster.service.MasterService$BatchingTaskQueue$Processor.lambda$run$2(MasterService.java:1691)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.action.ActionListener.run(ActionListener.java:452)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.cluster.service.MasterService$BatchingTaskQueue$Processor.run(MasterService.java:1688)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.cluster.service.MasterService$5.lambda$doRun$0(MasterService.java:1283)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.action.ActionListener.run(ActionListener.java:452)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.cluster.service.MasterService$5.doRun(MasterService.java:1262)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:1023)\r\n\tat org.elasticsearch.server@8.16.2-SNAPSHOT/org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:27)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\r\n\tat java.base/java.lang.Thread.run(Thread.java:1575)\r\n"},"status":400}]
```

**Issue Reasons:**
- [8.16] 2 failures in class org.elasticsearch.smoketest.DocsClientYamlTestSuiteIT (2.0% fail rate in 100 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/docs/build.gradle
+++ b/docs/build.gradle
@@ -130,8 +130,9 @@ testClusters.matching { it.name == "yamlRestTest"}.configureEach {
   setting 'xpack.security.enabled', 'true'
   setting 'xpack.security.authc.api_key.enabled', 'true'
   setting 'xpack.security.authc.token.enabled', 'true'
-  // disable the ILM history for doc tests to avoid potential lingering tasks that'd cause test flakiness
+  // disable the ILM and SLM history for doc tests to avoid potential lingering tasks that'd cause test flakiness
   setting 'indices.lifecycle.history_index_enabled', 'false'
+  setting 'slm.history_index_enabled', 'false'
   setting 'xpack.license.self_generated.type', 'trial'
   setting 'xpack.security.authc.realms.file.file.order', '0'
   setting 'xpack.security.authc.realms.native.native.order', '1'

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -358,8 +358,6 @@ tests:
     issue: https://github.com/elastic/elasticsearch/issues/117937
   - class: org.elasticsearch.xpack.security.authc.kerberos.KerberosAuthenticationIT
     issue: https://github.com/elastic/elasticsearch/issues/118414
-  - class: org.elasticsearch.smoketest.DocsClientYamlTestSuiteIT
-    issue: https://github.com/elastic/elasticsearch/issues/116788
 
 # Examples:
 #

